apps/mull-ui/src/app/pages/register/register.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { CustomTextInput } from '@mull/ui-lib';
+import * as Yup from 'yup';
+import { useFormik } from 'formik';
+import { Link } from 'react-router-dom';
+import { History } from 'history';
+import logo from '../../../assets/mull-logo.png';
+
+import './register.scss';
+
+export interface RegisterProps {
+  history: History;
+}
+
+const Register = ({ history }: RegisterProps) => {
+  const formik = useFormik({
+    initialValues: {
+      name: '',
+      email: '',
+      password: '',
+    },
+
+    validationSchema: Yup.object({
+      name: Yup.string().required('Name is required.'),
+      email: Yup.string().required('Email is required.').email('Email format is incorrect'),
+      password: Yup.string().required('Password is required.'),
+    }),
+
+    onSubmit: (values) => {},
+  });
+
+  return (
+    <form onSubmit={formik.handleSubmit}>
+      <div className="page-container-no-nav">
+        <div className="register-container">
```

Comment:
No need to have an extra div, you should be able to apply the styling to the same div.